py-polars/tests/unit/constructors/test_dataframe.py

```diff
@@ -123,9 +124,16 @@ def test_df_init_from_series_strict() -> None:
     assert df["a"].dtype == pl.UInt8
 
 
+# https://github.com/pola-rs/polars/issues/15471
 def test_df_init_rows_overrides_non_existing() -> None:
```

Comment:
```suggestion
def test_df_init_rows_overrides_non_existing_15471() -> None:
```

Reply:
Small difference but leaves to have more unique function names and less lines. :D

Reply:
Ok but hear me out 😄

1. The link is one click to get to the issue. With only the issue number, I have to figure out the URL to the actual issue somehow (I usually go to "Issues", click a random issue, replace the number). Too many steps!
2. If you give a proper name to the test it should already be unique. If it's not unique, either you have duplicate tests (if you find out, good, you can get rid of one) or the names are not descriptive enough.

A comment in the link is just more user friendly, imo. What do you think?

Reply:
Alright. Not a big deal to me. ^^